tests/installation/test_executor.py

```diff
@@ -165,19 +165,20 @@ def callback(
 
 
 @pytest.fixture
-def copy_wheel(tmp_dir: Path, fixture_dir: FixtureDirGetter) -> Callable[[], Path]:
+def copy_wheel(tmp_path: Path, fixture_dir: FixtureDirGetter) -> Callable[[], Path]:
     def _copy_wheel() -> Path:
         tmp_name = tempfile.mktemp()
-        Path(tmp_dir).joinpath(tmp_name).mkdir()
+        tmp_path.joinpath(tmp_name).mkdir()
 
         shutil.copyfile(
             (
                 fixture_dir("distributions") / "demo-0.1.2-py2.py3-none-any.whl"
             ).as_posix(),
-            (Path(tmp_dir) / tmp_name / "demo-0.1.2-py2.py3-none-any.whl").as_posix(),
+            tmp_path.joinpath(tmp_name)
+            .joinpath("demo-0.1.2-py2.py3-none-any.whl")
+            .as_posix(),
```

Comment:
```suggestion
            tmp_path / tmp_name / "demo-0.1.2-py2.py3-none-any.whl",
```

I suppose we can remove the `as_posix()` in the line above, too.